src/storage/unstructuredsoagrid.h

```diff
@@ -351,6 +355,35 @@ class UnstructuredSoAGrid : public GridBase<ELEMENT_TYPE, 1>
         elements.callback(&newGrid->elements, functor);
     }
 
+    void saveRegion(char *target, const Region<DIM>& region) const
```

Comment:
functions like these make me realize how incomplete unstructured grid support still is. good implementation and good thinking. thanks!